Use --no-show-signature option for git log

joelnb opened the pull request:
This should ensure that the pkgver function works as expected even for
people who have log.showSignature set to true in their git config.

I currently get this error when building these packages:

```
==> Starting pkgver()...
==> ERROR: pkgver is not allowed to contain colons, forward slashes, hyphens or whitespace.
==> ERROR: pkgver() generated an invalid version: r328.gpg: Signature made Tue 29 Oct 2019 14:59:49 GMT
gpg:                using RSA key 4AEE18F83AFDEB23
gpg: Good signature from "GitHub (web-flow commit signing) <noreply@github.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 5DE3 E050 9C47 EA3C F04A  42D3 4AEE 18F8 3AFD EB23
1572361189.29c6f0d
```

With this change I would not have to update my gitconfig or edit the PKGBUILD on every update.

nfnty commented:
Wow, that's some weird behavior on git's part. I really think this is an upstream bug. There's no way that the signature verification should be shown when a pretty format has been explicitly specified.

joelnb commented:
Yeah it is rather strange. It does it with `--oneline` too which does not look great:

```
410f3f7 (HEAD -> master, upstream/master, origin/master, origin/HEAD) gpg: Signature made Fri 25 Oct 2019 15:21:08 BST
gpg:                using RSA key 24565B8DDE064BE4FC62BACB32DD96A5D0370907
gpg: Good signature from "nfnty <git@nfnty.se>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2456 5B8D DE06 4BE4 FC62  BACB 32DD 96A5 D037 0907
Added `linux`
78b4667 gpg: Signature made Thu 24 Oct 2019 23:39:02 BST
gpg:                using RSA key 24565B8DDE064BE4FC62BACB32DD96A5D0370907
gpg: Good signature from "nfnty <git@nfnty.se>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2456 5B8D DE06 4BE4 FC62  BACB 32DD 96A5 D037 0907
Added `ovmf`
705bdde gpg: Signature made Fri 11 Oct 2019 11:10:12 BST
gpg:                using RSA key 24565B8DDE064BE4FC62BACB32DD96A5D0370907
gpg: Good signature from "nfnty <git@nfnty.se>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2456 5B8D DE06 4BE4 FC62  BACB 32DD 96A5 D037 0907
Updated
...
```
